prow/config.yaml

```diff
@@ -988,6 +1128,103 @@ presubmits:
         - name: cache-ssd
           hostPath:
             path: /mnt/disks/ssd0
+  - name: pull-security-kubernetes-bazel-build
+    agent: kubernetes
+    context: pull-security-kubernetes-bazel-build
+    always_run: true
+    rerun_command: "/test pull-security-kubernetes-bazel-build"
+    trigger: "(?m)^/test( all| pull-security-kubernetes-bazel-build),?(\\s+|$)"
+    branches:
+    - release-1.6
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/bazelbuild:v20170929-ed26bd34-0.5.2
```

Comment:
is this image too old? I thought we fixed this issue:
```
W1011 00:04:16.477] Error copying file:///workspace/k8s.io/kubernetes/_artifacts: CommandException: Encountered OSError running "file --mime-type /workspace/k8s.io/kubernetes/_artifacts/junit_bazel.xml"
```

Reply:
Hmm, We should definitely make it match the others.

Reply:
Actually it looks like they match already.

Reply:
https://github.com/kubernetes/test-infra/blob/d6ec1d17746c28650a7f7c25f11461894d84e9ff/prow/config.yaml#L55-L67